cmd/deploy/deploy.go

```diff
@@ -623,3 +608,32 @@ func (dc *DeployCommand) trackDeploy(manifest *model.Manifest, runInRemoteFlag b
 		IsRemote:               isRunningOnRemoteDeployer,
 	})
 }
+
+func checkOktetoManifestPathFlag(options *Options) error {
+	if options.ManifestPath != "" {
```

Comment:
Exiting early improves overall readability, I would do:

```go
if options.ManifestPath == "" {
	return nil
}
```

Reply:
when the manifest path is not set, why are we returning the error as nil? shouldn't we return an error something like "the manifest path doesn't exist" ?

Reply:
@nitishfy sorry for the late reply, absolutely, I would actually remove this check from the function because it should go before this function is even called. We should only validate the path if defined

Reply:
Here's one more thing I observed:

The above piece of code you wrote will work if use the command `okteto deploy`. Even if we use this piece of code before the function call, the command `okteto deploy` will always lead to error. In reality, `okteto deploy` is designed to work without even mentioning the path. I think we can remove this check since everything is working fine already. WDYT?